src/service/performance-impl.js

```diff
@@ -97,7 +97,7 @@ export class Performance {
         });
 
     // Tick window.onload event.
-    loadPromise(win).then(() => {
+    loadPromise(win.document).then(() => {
```

Comment:
I'm trying to understand why we are using `window.onload` vs a more predictable `onDocumentReady`?